Samples/RenderGraphViewer/RenderGraphViewer.cpp

```diff
@@ -97,12 +132,36 @@ void RenderGraphViewer::loadScene(const std::string& filename, bool showProgress
     mSceneFilename = filename;
     mCamControl.attachCamera(mpScene->getCamera(0));
     mpScene->getActiveCamera()->setAspectRatio((float)pSample->getCurrentFbo()->getWidth() / (float)pSample->getCurrentFbo()->getHeight());
-    createGraph(pSample);
 }
 
 void RenderGraphViewer::onLoad(SampleCallbacks* pSample, const RenderContext::SharedPtr& pRenderContext)
 {
+#ifdef _WIN32
+    // if editor opened from running render graph, get memory view for live update
+    std::string commandLine(GetCommandLineA());
```

Comment:
Nope. No WIN32 calls here.
You can use ArgList to get an already parsed list of arguments